dev-client/src/components/IntervalForm.tsx

```diff
@@ -49,11 +48,10 @@ export const IntervalForm = () => {
           })}
         </FormControl.HelperText>
       </FormControl>
-      <Row justifyContent="space-between" space="40px">
+      <Row justifyContent="space-between" space="20px" pt="10px">
```

Comment:
I believe the conversion from "unitless nativebase units" to px is *4 ... so I suspect you want 
```suggestion
      <Row justifyContent="space-between" space="40px" pt="20px">
```

Slight difference between the nativebase units you had before, and the px you have now:
<img width="798" alt="Screenshot 2024-06-07 at 5 21 30 PM" src="https://github.com/techmatters/terraso-mobile-client/assets/5590815/88d14672-5d7a-4c56-b8cb-7a00e219658e">